build-packages/scandipwa-cli/actions/extension/index.js

```diff
@@ -1,143 +1,79 @@
-    );
-
-    try {
-        // generate extension
-        await generateExtension({
-            name: packageName,
-            path: packagePath
-        });
-    } catch (e) {
-        logger.log(e);
-
-        logger.error(
-            `Failed to create ScandiPWA extension in ${ logger.style.file(packagePath) }.`,
-            'See the error log above.'
-        );
-
-        process.exit();
-    }
-
-    try {
-        // add package as dependency and install sub-dependencies
-        await injectScripts(contextPathname);
-        await addDependency(contextPathname, packageName, `file:${ relativePackagePath }`);
-        await installDeps(contextPathname);
-    } catch (e) {
-        logger.log(e);
-
-        logger.error(
-            `Failed to install ScandiPWA extension in ${ logger.style.file(contextPathname) }.`,
-            'See the error log above.'
-        );
-
-        process.exit();
-    }
-
-    if (enable) {
-        enableExtension(contextPathname, packageName);
-    }
-};
-
-const installExtension = async (argv) => {
-    const {
-        enable = true,
-        package: packageName
-    } = argv;
-
-    const { type: context, pathname: contextPathname } = walkDirectoryUp(process.cwd(), THEME_TYPE);
-
-    if (!context) {
-        // make sure we are in ScandiPWA theme context
-        logger.error(
-            'To create an extension you must be located in ScandiPWA theme directory.',
-            `We looked up six folders up starting from ${ logger.style.file(process.cwd()) }!`,
-            `There was no folders containing ${ logger.style.file('package.json') }, where ${ logger.style.misc('scandipwa.type') } field was equal to ${ logger.style.misc('theme') }.`
-        );
-
-        process.exit();
-    }
-
-    try {
-        // add package as dependency and install sub-dependencies
-        await addDependency(contextPathname, packageName);
-        await installDeps(contextPathname);
-    } catch (e) {
-        logger.log(e);
-
-        logger.error(
-            `Failed to install ScandiPWA extension in ${ logger.style.file(contextPathname) }.`,
-            'See the error log above.'
-        );
-
-        process.exit();
-    }
-
-    if (enable) {
-        enableExtension(contextPathname, packageName);
-    }
-};
 
 module.exports = (yargs) => {
     yargs.command('extension <command>', 'Interact with extension', (yargs) => {
-        yargs.command('install <package>', 'Install and enable ScandiPWA extension', (yargs) => {
+        yargs.command('install <name>', 'Install and enable ScandiPWA extension', (yargs) => {
             yargs.option('no-enable', {
                 describe: 'Do not enable installed extension.'
             });
-        }, installExtension);
+
+            yargs.option('local', {
+                alias: 'l',
+                describe: `Use a local module from ${logger.style.file('packages/<name>')}`,
+                type: 'boolean',
+                default: false
+            });
```

Comment:
Why this exists, if there is one very simmilar `-u`?

Reply:
This is a shortcut for `-u`, to use the default `packages/...` directory. Just for convenience :)